### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by k-rahul21**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+.brand-name:hover {
+  color: var(--secondary-color);
+}
+
+.list-non-bullet {
+  list-style: none;
+}
+
+.f-bold {
```
Comment (bharati-21) [inline css/main.css]: f-bold seems like an unusual class name. You could just add font-bol.d

**Review by tewarimohit — APPROVED** [review]
Looks good to merge...

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+main .subhead {
+  width: 90%;
+  max-width: 750px;
+  font-size: 1.2rem;
+  line-height: 2;
+  margin-bottom: 1.5rem;
+}
+
+@media screen and (max-width: 800px) {
```
Comment (bharati-21) [inline css/main.css]: Suggest to add all media queries together at the end of the file.

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+.btn {
+  font-weight: 600;
+  color: var(--text-color);
+  min-width: 100px;
+  padding: 0.8rem;
+  border-radius: 0.2rem;
+  border: none;
+  margin: 1rem;
+  box-shadow: rgb(0 0 0 / 24%) 0px 4px 9px;
```
Comment (bharati-21) [inline css/main.css:104]: You could be consistent and use either variables or colors everywhere,

File: index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (bharati-21) [inline index.html:8]: Not important, but you can add the meta author and description tags.

**Review by bharati-21 — COMMENTED** [review]
Just dropped a few suggestions that you could follow if you have the bandwidth. 
The other code seems to be fine. 👍🏽

**Comment by bharati-21** [comment]
> _ No description provided. _

Also, please do provide a description and preview so that it becomes easier for reviewers 👍🏽

**Pull request merged**